optaplanner-benchmark/src/main/java/org/optaplanner/benchmark/impl/report/BenchmarkReport.java

```diff
@@ -529,6 +549,64 @@ private void writeBestScorePerTimeSpentSummaryChart() {
         }
     }
 
+    private void writeSubSingleBenchmarkScoreCharts() {
```

Comment:
Agreed to leave it here (I don't see a better place either).
As for PLANNER-443, we 'll cross that bridge when we get there.